back/engines/free/email_campaigns/app/models/concerns/email_campaigns/schedulable.rb

```diff
@@ -37,5 +37,16 @@ def force_schedule_start_in_config_timezone
       ics.start_time = ics.start_time.in_time_zone(AppConfiguration.instance.settings('core', 'timezone'))
       self.ic_schedule = ics
     end
+
+    def schedule_multiloc_value(locale)
+      # We currently only support weekly schedules here.
+      weekly_day_key = "email_campaigns.schedules.weekly.#{schedule['rrules'][0]['validations']['day'][0].to_i % 7}"
+      hour = schedule['rrules'][0]['validations']['hour_of_day'][0].to_i
+
+      hour_of_day = "#{hour}:00"
+      hour_of_day = hour > 12 ? "#{hour % 12}.00 PM" : "#{hour}.00 AM" if %w[en en-GB en-US en-CA].include?(locale)
```

Comment:
Unfortunately Il8n doesn't help us much with deciding localized time formats, so we're going to need something like this (not necessarily here).

Alternatively, we could specify the formats in specific YAML language files, which seems overkill for so few languages using 12-hour format (and most of these could cope with 24-hour format anyway). Though, I still need to research some languages for preferred formats, eg. Arabic languages)